utils/build-script-impl

```diff
@@ -2039,6 +2051,15 @@ for host in "${ALL_HOSTS[@]}"; do
                     )
                 fi
 
+                if contains_product "lldb" ; then
```

Comment:
Ahh, much appreciated @rintaro, my f$@#-up. Could that be the source of my issues ... Retesting (I have to figure out how the hell my local tests worked at all!).